protos/service_mesh.proto

```diff
@@ -11,7 +11,7 @@ option objc_class_prefix = "smp";
 option php_namespace = "Smp";
 option ruby_package = "Smp";
 
-message ServiceMeshes {
+message ServiceMesh {
```

Comment:
@kumarabd Any thoughts on changing `ServicesMeshes` to `ServiceMesh`. Am still conflicting with the idea.

Reply:
Good reference - https://developers.google.com/protocol-buffers/docs/style

Reply:
ServiceMesh it is